ScreenManager.lua

```diff
@@ -533,6 +528,7 @@ end
 --
 function ScreenManager.registerCallbacks( callbacks )
     local registry = {};
+    local function null() end
```

Comment:
Nah, in Screen it just looked strange to me to have it on one line. Inside of the function it's ok like this :)